README.md

```diff
@@ -94,13 +99,15 @@ The tool has been designed specifically to run using docker swarm or compose but
 ```
 https://github.com/uccser/swarm-stack-updater.git
 ```
-2. Use the following command to pull BATS, which is used for automated testing. This will pull all the required tools for running and creating tests for this application.
+2. Use the following command to pull BATS, which is used for automated testing.
+  This will pull all the required tools for running and creating tests for this application.
 ```
 git submodule update --init
 ```
 3. Ensure that you have a configuration file created and have the requried environment varibles defined. To define environment varibles use the command ```export ENV_VAR=<value>```.
 
-4. Run the swarm updater by running the command ```./src/swarm_stack_updater.sh.``` You may need to set to to be executible this can be done by typing the command ```chmod +x /src/swarm_stack_updater.sh```
+4. Run the swarm updater by running the command ```./src/swarm_stack_updater.sh```.
+  You may need to set to to be executible this can be done by typing the command ```chmod +x /src/swarm_stack_updater.sh```
```

Comment:
Probably best to make these inline code blocks (single back delimiters) to match the ones in the following lines.